test/RelayHubPenalizations.test.js

```diff
@@ -11,13 +11,15 @@ const rlp = require('rlp')
 
 const { expect } = require('chai')
 
-contract('RelayHub Penalizations', function ([_, relayOwner, relay, otherRelay, sender, other]) { // eslint-disable-line no-unused-vars
+const GTXDATANONZERO = 68
+
+contract.skip('RelayHub Penalizations', function ([_, relayOwner, relay, otherRelay, sender, other]) { // eslint-disable-line no-unused-vars
```

Comment:
Why is it skipped?

Reply:
Because tests fail for me locally (EIP-155 issue) and I don't want to run these tests even if they pass on CircleCI without fixing it, as they depend solely on having a very old ganache to pass.

Reply:
what "old ganache"? we `yarn install` our ganache version.
either show why this method is redundant with another test - or let's fix it.

Reply:
We yarn install the old ganache version. I can enable these tests on CircleCI, but they will likely fail locally if you run a different ganache. I don't want tests flickering depending on the ganache version in the repo.
This "method" is not redundant. This "method" is all tests of penalizations. 
Penalizations are completely broken as described in detail.